app.js not completed yet

gamboagirl opened the pull request:
This is what I have so far. I created the package.json, and updated trucks.js as well. I am working on app.js, but working through errors. I'll update when I can get it working.

gamboagirl committed:
Adds package.json and updated trucks.js

Trucks.js is updated with functions, but not yet tested.

gamboagirl committed:
Add updates to file

Updates apps.js and client .js and includes commented out trucks.js.
Syncing changes so they're accessible from my other computer.

gamboagirl committed:
Adds more to the app.js

jamdon commented on app.js:
```diff
@@ -2,3 +2,58 @@
+
+app.use(express.static('public'));
+
+app.get('/', function (request, response) {
```
This is unnecessary. Using express.static as middle-ware processes the root route so this code won't do anything.

jamdon commented on app.js:
```diff
@@ -2,3 +2,58 @@
+    var list;
+    
+    if (truck.name) {
+        list += "<li>Name: " + truck.name;
```
Browsers are forgiving with markup, but you should add the closing li tag when constructing your markup.

jamdon commented:
It's looking pretty good so far.

gamboagirl committed:
Updates app.js, client.js, and trucks.js

The difficult part wasn't the app.get calls, it was understanding the
underscore functions and how to get it work on the data. _.flatten is
one I had to study to understand what it really does.

gamboagirl commented:
I updated the files and synced, but it doesn't seem like I can create a new pull request. Does it automatically update with my new commits?

jamdon commented:
Yes, it automatically updates because the pull request is simply linking back to your repository as opposed to a snapshot of your repository when you initiated the pull request.

jamdon commented:
:+1:
